### ISSUE TITLE
quick_error macro emits "doc comment not used by rustdoc"

### ISSUE THREAD

**Issue by Marwes** [tailhook/quick-error#39]
Putting doc comments on the variants in the quick_error macro causes rustc to emit warnings despite the comments actually appearing in rustdoc. https://docs.rs/gluon/0.5.0/gluon/enum.Error.html

```rust
quick_error! {
    #[derive(Debug, PartialEq)]
    pub enum Error {
        /// Some comment
        Error(err: String) {
            display("{}", err)
        }
    }
}
```

I guess this is because the doc comments/attributes gets attached to something that rustdoc does not use in addition to some place which it does use. Would it be possible to supress the warnings in the macro itself or should it be supressed on the call-site?

**Comment by tailhook** [comment]
Sorry, I don't see the warning. Any way to reproduce it? Is it for some older rust version? Or should I enable any extra warnings?

**Comment by Marwes** [comment]
Did you test it on nightly? Should have mentioned that it is a new warning.

**Comment by tailhook** [comment]
Ah, now I see. Added suppression attributes. Should be fixed now, will upload the update on crates.io soon.

**Comment by tailhook** [comment]
Released as `v1.2.1`
